src/components/core/Main/Main.module.scss

```diff
@@ -8,3 +8,7 @@
 .title {
   color: $gray;
 }
+
+.headline {
+  color: green;
```

Comment:
I recommend using a variable with colors.
You can add a new variable in file `colors.scss` and using it.